lib/src/index.node.ts

```diff
@@ -0,0 +1,13 @@
+import { NanoTDFClient, NanoTDFDatasetClient, AuthProviders, version, clientType } from './index';
+import fetch from 'node-fetch';
```

Comment:
Yeah, it looks like node-fetch is the way to go:

https://www.npmtrends.com/cross-fetch-vs-isomorphic-fetch-vs-node-fetch-vs-whatwg-fetch